.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,57 +1,131 @@
-<!--  Thanks for sending a pull request!  Here are some tips for you:
+<!--  
+Thanks for submitting a pull request!
 
-1. If this is your first time, please read our contributor guidelines: https://git.k8s.io/community/contributors/guide#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
-2. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
-https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
-3. Ensure you have added or ran the appropriate tests for your PR: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
-4. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
-5. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
-6. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+If this is your first time submitting a PR, please familiarise yourself with:
+
+  Contributor Guidelines 
+  https://git.k8s.io/community/contributors/guide#your-first-contribution
+
+  Developer Guide 
+  https://git.k8s.io/community/contributors/devel/development.md#development-guide
+
+The following are additional useful links for PRs:
+
+  Best Practices for Faster Reviews 
+  https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
+
+  Marking Unfinished PRs for Review or as a Work in Progress (WIP)
+  https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+-->
+
+**Which is the sponsoring or primary SIG associated with this PR?**
+<!--
+Label the sponsoring or primary SIG associated with this PR.
+
+For reference on available SIGs, you can find more details at:
+https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md#labels-that-apply-to-all-repos-for-both-issues-and-prs
+
+Add only ONE sig here via a /sig-primary <name> entry.
 -->
+/sig-primary ???
+
+> DISCUSSION REQUIRED
+>
+> Should we use the above pattern and update the label prow plugin ? https://github.com/kubernetes/test-infra/tree/master/prow/plugins/label. Logic would have to ensure that there is only a single primary sig associated. This metadata can be leveraged by other processes outside of the release notes.
+> 
+> or, if this is ONLY relevant to release notes then, we can add here as a `` ```primary-sig`` code block.
```

Comment:
Yes. This is only relevant for release notes.
With this field filled in, we can adust the relnotes tool to collect the primary SIG so that the relnote associated with the PR can be sorted as the PR author intended (saving a lot of stress and time for the relnotes team and contributors frantically editing the relnotes in the days just prior to the release :joy: ). All SIG labels would remain as before and would be searchable via  relnotes.k8s.io

Reply:
So I'll leave the `/sig-primary` section in for now, and start looking at all the dependencies - like the label prow plugin.

I'll update the PR comments with dependencies based on some final decisions we make.